packages/webapp/pages/squads/index.tsx

```diff
@@ -60,6 +64,8 @@ const SquadsPage = ({ initialData }: Props): ReactElement => {
     },
   );
 
+  if (!isAuthReady || (user && !isActionsFetched)) return null;
```

Comment:
The `isActionsFetched` is always `false` when it is for anonymous user as the query fails.

Reply:
I think it's a bad practice to block the rendering altogether as it will interrupt the seo aspects of the page. I suggest conditionally rendering the relevant parts

Reply:
Makes sense 😊